src/handler.js

```diff
@@ -3,27 +3,33 @@ const httpStatus = require('http-status-codes')
 const logger = require('pino')({ useLevelLabels: true })
 
 logger.info('Logger initialized')
-exports.handler = async (event, context) => generateProblemResponse(event, httpStatus.INTERNAL_SERVER_ERROR)
 
-const generateProblemResponse = (event, errorCode) => {
-  const reason = httpStatus.getReasonPhrase(errorCode)
-  const detail = `Your request cannot be processed at this time due of '${reason}'`
-  const path = event.path
-  logger.info(detail, path)
+const routes = []
+
+exports.handler = async (event, context) => {
```

Comment:
Suggestion:
```
exports.handler = async (event, context) => {
  const responseDetails = respondNotFoundForAllPaths(event)
  return errorResponse(responseDetails, event)
}


function requestContainsPath(event) {
  return 'path' in event && !routes.includes(event.path);
}

function notFoundResponseDetails(event) {
  return {
    code: httpStatus.NOT_FOUND,
    message: `The requested resource ${event.path} could not be found`
  };
}

function internalServerErrorResponseDetails() {
  return {
    code: httpStatus.INTERNAL_SERVER_ERROR,
    message: 'Your request cannot be processed at this time due to an internal server error'
  };
}

function respondNotFoundForAllPaths(event) {
  return requestContainsPath(event)
      ? notFoundResponseDetails(event)
      : internalServerErrorResponseDetails();
}
```
My suggestion is basically to show the intention of the present code which is to return Not Found for all unknown paths and that presently all paths are unknown. 

Regarding function declarations and function expressions I would suggest to follow the "newspaper" pattern, i.e., we go from the more general code on the top (the handler function) to more specific details (towards the bottom). According to documentation arrow functions cannot be hoisted and based on that I would suggest to use function declarations.  But, when I run the code using function expressions the tests are green, so I am not sure whether it actually matters or not.